src/find_ice_labels.jl

```diff
@@ -21,7 +21,7 @@ function find_reflectance_peaks(
     reflectance_channel::Union{__T__,Matrix{Float64}};
     possible_ice_threshold::Float64=Float64(75 / 255),
 )::Int64
-    reflectance_channel[reflectance_channel .< possible_ice_threshold] .= 0 #75 / 255
+    reflectance_channel[reflectance_channel.<possible_ice_threshold] .= 0 #75 / 255
```

Comment:
What's the reasoning behind taking out the white spaces here, but not around the `.=`? Is this a standard Julia style? To me the white spaces help with readability.